### PR TITLE
Add info re: sshd config with DM workers

### PR DESCRIPTION
**Pull request by bdevcich**
Signed-off-by: Blake Devcich <blake.devcich@hpe.com>

### CODE REVIEW

File: docs/guides/data-movement/readme.md [hunk 1]
```diff
@@ -7,20 +7,18 @@ categories: provisioning
...
 
-## Server Side ConfigMap
+## `NnfDataMovementProfiles`
 
-The server side configuration is done via the `nnf-dm-config` config map:
-
-```bash
-kubectl -n nnf-dm-system get configmap nnf-dm-config
-```
+The server side configuration is done by creating `NnfDataMovementProfiles` resources in kubernetes.
```
Comment (ajfloeder) [inline docs/guides/data-movement/readme.md:20]: "is done" -> "is controlled"?

File: docs/guides/data-movement/readme.md [hunk 2]
```diff
@@ -7,20 +7,18 @@ categories: provisioning
...
+These work similar to `NnfStorageProfiles` #TODO LINK.
 
 The config map allows you to configure the following:
```
Comment (ajfloeder) [inline docs/guides/data-movement/readme.md:23]: Do you still have a config map?

File: docs/guides/data-movement/readme.md [hunk 3]
```diff
@@ -90,12 +88,19 @@ The `CreateRequest` API call that is used to create Data Movement with the Copy
...
 are on a per-request basis.
 
-The Copy Offload API requires the `nnf-dm` daemon to be running on the compute node. This daemon may be configured to run full-time, or it may be left in a disabled state if the WLM is expected to run it only when a user requests it. See [Compute Daemons](../compute-daemons/readme.md) for the systemd service configuration of the daemon. See `RequiredDaemons` in [Directive Breakdown](../directive-breakdown/readme.md) for a description of how the user may request the daemon, in the case where the WLM will run it only on demand.
+The Copy Offload API requires the `nnf-dm` daemon to be running on the compute node. This daemon may
+be configured to run full-time, or it may be left in a disabled state if the WLM is expected to run
+it only when a user requests it. See [Compute Daemons](../compute-daemons/readme.md) for the systemd
+service configuration of the daemon. See `RequiredDaemons` in [Directive
+Breakdown](../directive-breakdown/readme.md) for a description of how the user may request the
+daemon, in the case where the WLM will run it only on demand.
```
Comment (ajfloeder) [inline docs/guides/data-movement/readme.md:96]:
```suggestion
daemon in the case where the WLM will run it only on demand.
```

File: docs/guides/data-movement/readme.md [hunk 4]
```diff
@@ -115,3 +120,13 @@ option.
 
 See the [`dcp` documentation](https://mpifileutils.readthedocs.io/en/latest/dcp.1.html) for more
 information.
+
+## `sshd` Configuration for Data Movement Workers
+
+The nnf-dm-worker pods run `sshd` in order to listen for `mpirun` jobs to perform data movement. The
+number of simultaneous connections is limited via the sshd configuration (i.e. `MaxStartups`). **If
+you see error messages in Data Movement where mpirun can not communication with target nodes, this
```
Comment (ajfloeder) [inline docs/guides/data-movement/readme.md:128]:
```suggestion
you see error messages in Data Movement where mpirun cannot communicate with target nodes, this
```

**Review by ajfloeder — APPROVED** [review]

File: docs/guides/data-movement/readme.md [hunk 5]
```diff
@@ -90,12 +88,19 @@ The `CreateRequest` API call that is used to create Data Movement with the Copy
...
+The Copy Offload API requires the `nnf-dm` daemon to be running on the compute node. This daemon may
+be configured to run full-time, or it may be left in a disabled state if the WLM is expected to run
+it only when a user requests it. See [Compute Daemons](../compute-daemons/readme.md) for the systemd
+service configuration of the daemon. See `RequiredDaemons` in [Directive
+Breakdown](../directive-breakdown/readme.md) for a description of how the user may request the
+daemon, in the case where the WLM will run it only on demand.
 
-If the WLM is running the `nnf-dm` daemon only on demand, then the user can request that the daemon be running for their job by specifying `requires=copy-offload` in their `DW` directive. The following is an example:
+If the WLM is running the `nnf-dm` daemon only on demand, then the user can request that the daemon
```
Comment (roehrich-hpe) [inline docs/guides/data-movement/readme.md:98]: Strike this paragraph. In the paragraph above you've already told them where to look for this info.

I'm not so keen on maintaining the same info in 3 different docs.

File: docs/guides/data-movement/readme.md [hunk 6]
```diff
@@ -7,20 +7,18 @@ categories: provisioning
...
 Data Movement can be configured in multiple ways:
 
-1. Server side
+1. Server side (`NnfDataMovementProfile`)
 2. Per Copy Offload API Request arguments
 
-The first method is a "global" configuration - it affects all data movement operations. The second
-is done per the Copy Offload API, which allows for some configuration on a per-case basis, but is
-limited in scope. Both methods are meant to work in tandem.
+The first method is a "global" configuration - it affects all data movement operations that use a a
```
Comment (roehrich-hpe) [inline docs/guides/data-movement/readme.md:13]:
```suggestion
The first method is a "global" configuration - it affects all data movement operations that use a
```

**Review by matthew-richerson — APPROVED** [review]

File: docs/guides/data-movement/readme.md (hunk 2 above)
Comment (bdevcich) [reply docs/guides/data-movement/readme.md:23]: I think I popped some changed before I committed and didn't notice. I'll clean this up.

**Comment by bdevcich** [comment]
Let's try this again. I did some cleanup and reorganized a bit, so please take another look.

**Review by roehrich-hpe — APPROVED** [review]

**Review by ajfloeder — APPROVED** [review]

**Review by matthew-richerson — APPROVED** [review]

**Pull request merged**